asset.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"gorm.io/gorm"
+)
+
+type Asset struct {
+	ID       uint           `gorm:"primaryKey;column:id;autoIncrement"`
+	AssetID  string         `gorm:"column:asset_id;index"` // "usdc" - internal unified asset id
+	Token    common.Address `gorm:"column:token;not null;uniqueIndex:token_chain"`
+	ChainID  uint32         `gorm:"column:chain_id;not null;uniqueIndex:token_chain"`
+	Symbol   string         `gorm:"column:symbol;not null"`
+	Decimals uint8          `gorm:"column:decimals;not null"`
+}
```

Comment:
Let's split assets and tokens, so we can manage the difference in decimal points of the same token on different chains